### ISSUE TITLE
ceterms:ActionStatus ConceptScheme should not be owl:equivalentClass to schema:ActionStatus

### ISSUE THREAD

**Issue by philbarker** [CredentialEngine/Schema-Development#959]
For [ceterms:ActionStatus](https://credreg.net/ctdl/terms#ActionStatus) we have:
`              owl:equivalentClass schema:ActionStatus ;`

Two problems:
- There is no https://schema.org/ActionStatus class. There is a https://schema.org/actionStatus property and a https://schema.org/ActionStatusType enumeration Class.
-  It is questionable whether an rdfs:Class can be equivalent to a skos:ConceptScheme (even if the members are the same)—I was talking to one of the authors of SKOS last week, he called it "unorthodox to say the least".  
 
I think schema:isBasedOn would be a better term to express the relationship, i.e.
`ceterms:ActionStatus schema:isBasedOn schema:ActionStatusType`

**Comment by siuc-nate** [comment]
Is the fix just to remove the equivalency then, or does this need more discussion?

**Comment by philbarker** [comment]
Can you replace the owl:equivalentClass with schema:isBasedOn?

**Comment by siuc-nate** [comment]
I could, but what would it point to?

**Comment by philbarker** [comment]
ceterms:ActionStatus schema:isBasedOn schema:ActionStatusType

**Comment by philbarker** [comment]
Proposal:

**Modify Existing Concept Scheme**

>  **URI:** ceterms:ActionStatus
>  **Remove** equivalentClass schema:ActionStatus
>  **Add** isBasedOn  schema:ActionStatus

OK?

**Comment by philbarker** [comment]
@siuc-nate is it possible to add that as a predicate  for terms we create? I scanned through the Schema Updates doc and didn't see it, but I guess it is a code level change?

**Comment by siuc-nate** [comment]
Correct, it would require a code change. I created a task in our internal tracker for it.

**Comment by philbarker** [comment]
For now we can:

**URI:** ceterms:ActionStatus

> **Remove isBasedOn** equivalentClass schema:ActionStatus

**Comment by philbarker** [comment]
The following was entered in the schema management system for the Dec 2025 release, and noted in the history tracking:

**URI:** ceterms:ActionStatus

> **Remove equivalentClass**  schema:ActionStatus


This removes the incorrect statement. We should enter the correct acknowledgement when we can.

**Comment by philbarker** [comment]
Completed in December 2025 release.
